plugins/home/src/api/VisitsApi.ts

```diff
@@ -65,24 +57,36 @@ export type VisitsApiQueryParams = {
    */
   limit?: number;
   /**
-   * A record for which the key is a field name to sort on, and the value is the sort direction.
-   * For a multi-field sorting query, add multi entries to the record.
+   * Allows ordering visits on entity properties.
    * @example
    * Sort ascending by the timestamp field.
    * ```
-   * { orderBy: { timestamp: 'asc' } }
+   * { orderBy: [{ field: 'timestamp', direction: 'asc' }] }
    * ```
    */
-  orderBy?: Record<string, 'asc' | 'desc'>;
+  orderBy?: Array<{
+    field: keyof Visit;
+    direction: 'asc' | 'desc';
+  }>;
   /**
-   * Allows filtering visits on number of hits, timestamp and/or entityRef attributes.
+   * Allows filtering visits on entity properties.
    * @example
    * Most popular docs on the past 7 days
    * ```
-   * { orderBy: { hits: 'desc' }, filterBy: [{ field: 'timestamp', operator: '>=', value: <date> }, { field: 'entityRef', operator: 'contains', value: 'docs' }] }
+   * {
+   *   orderBy: [{ field: 'hits', direction: 'desc' }],
+   *   filterBy: [
+   *     { field: 'timestamp', operator: '>=', value: <date> },
+   *     { field: 'entityRef', operator: 'contains', value: 'docs' }
+   *   ]
+   * }
    * ```
    */
-  filterBy?: VisitFilter[];
```

Comment:
Necessary disclaimer. I've altered this under the presumption that Javascript object attribute order was unpredictable, I was [wrong](https://262.ecma-international.org/6.0/#sec-ordinary-object-internal-methods-and-internal-slots-ownpropertykeys).
Another viable alternative could look like:
```
{
  orderBy: { hits: { direction: 'desc' }  },
  filterBy: {
    timestamp: { operator: '>=', value: <data> },
    entityRef: { operator: 'contains', value: 'docs' },
  }
}
```
I think it's important to keep consistency between both. I prefer the current version because it is explicit about the fact that the order the parameters are passed will affect the result. It does not rely on knowing implementation details of JS specifications.